molecule/utilities.py

```diff
@@ -24,9 +24,14 @@
 import os
 import sys
 
+import logging
+
 import colorama
 import jinja2
 
+log = logging.getLogger()
+log.addHandler(logging.NullHandler())
```

Comment:
Maybe also renaming ``log`` with ``logger`` (nitpicking I know, but it sound clearer)